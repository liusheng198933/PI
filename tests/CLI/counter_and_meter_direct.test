#!/bin/sh
./run_one_test.py $srcdir/testdata counter_and_meter_direct stats.json
