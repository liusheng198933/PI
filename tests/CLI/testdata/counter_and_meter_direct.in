counter_write ExactOne_counter NEXT_ENTRY bytes=388
meter_set ExactOne_meter NEXT_ENTRY 1:1 2:3
table_add ExactOne 0 => actionA 0
counter_read ExactOne_counter 0
meter_read_spec ExactOne_meter 0
counter_write ExactOne_counter NEXT_ENTRY bytes=99 packets=2
table_modify ExactOne 0 => actionA 0
counter_read ExactOne_counter 0
meter_read_spec ExactOne_meter 0
