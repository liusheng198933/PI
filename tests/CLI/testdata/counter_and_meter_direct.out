????
Device assigned successfully.
Selecting device.
????
Dumping counter entry:
	Bytes: 388
Direct resource spec was stored.
????
Dumping meter spec:
	Meter unit: bytes
	Meter type: color unaware
	Committed information rate: 1
	Committed burst size: 1
	Peak information rate: 2
	Peak burst size: 3
Direct resource spec was stored.
????
Entry was successfully added with handle 0.
????
Dumping counter entry:
	Packets: 0
	Bytes: 388
????
Dumping meter spec:
	Meter unit: bytes
	Meter type: color unaware
	Committed information rate: 1
	Committed burst size: 1
	Peak information rate: 2
	Peak burst size: 3
????
Dumping counter entry:
	Packets: 2
	Bytes: 99
Direct resource spec was stored.
????
Entry with handle 0 was successfully modified.
????
Dumping counter entry:
	Packets: 2
	Bytes: 99
????
Dumping meter spec:
	Meter unit: bytes
	Meter type: color unaware
	Committed information rate: 1
	Committed burst size: 1
	Peak information rate: 2
	Peak burst size: 3
????
